app/src/main/java/ch/epfl/sweng/hostme/ui/search/DisplayApartment.java

```diff
@@ -73,6 +58,12 @@ public DisplayApartment() {
     public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
         root = inflater.inflate(R.layout.display_apartment, container, false);
+
+        Button maps_button = root.findViewById(R.id.maps_button);
+        maps_button.setOnClickListener(this::goToMapsFragment);
+        Button street_view_button = root.findViewById(R.id.street_view_button);
+        street_view_button.setOnClickListener(this::goToStreetViewFragment);
```

Comment:
I see that you use `snake_case` for a lot of UI components. Personally, I have been using `camelCase` for all of my variables except `CAPITALIZED_CONSTANTS`. Maybe some time we can have a quick discussion to choose one as a standard, then I can make it consistent across the repo.

Reply:
yes of course we can see that together at the next meeting